packages/server/test/support/fixtures/projects/ts-proj/cypress/integration/app_spec.ts

```diff
@@ -1,3 +1,4 @@
+import { expect } from 'chai'
```

Comment:
is this line expected?

Reply:
Yes, it's okay in this case to use a local version of chai expect since the test doesn't need anything cypress adds to its global version.

Reply:
```suggestion
```